src/interpreter/bytecode.h

```diff
@@ -131,6 +132,8 @@ namespace verona::bytecode
     uint32_t size;
   };
 
+  constexpr static uint32_t VERONA_NUMBER = 4085854915;
```

Comment:
Hex notation is better for this, so it's clear all 4 bytes are used, and it gives something to look for when hexdump'ing a bytecode file.

Should be called `MAGIC_NUMBER` instead, as verona number is too vague.